addons/main_a3/CfgMods.hpp

```diff
@@ -2,15 +2,13 @@ class CfgMods {
     class PREFIX {
         author = "$STR_CBA_Author";
         dir = "@CBA_A3";
-        name = "Community Base Addons v3.9.1";
```

Comment:
change it to 0.0.0
there is code to change on build
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L77
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L597

Reply:
```suggestion
        name = "Community Base Addons v0.0.0";
```
Like this?

Reply:
yeah, also need to change
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L77
versionFiles = ["mod.cpp","addons\main_a3\CfgMods.hpp"]